react-with-graphql-tutorial.md

```diff
@@ -1522,11 +1525,9 @@ const GET_ISSUES_OF_REPOSITORY = `
 `;
 {{< /highlight >}}
 
-In the previous template string, the `cursor` is passed as variable to the query and used as `after` argument for the list field. The variable is not enforced though, because there is no exclamation mark next to it. Thus, it can be `undefined` which happens for the initial page requests for a paginated list where you only want to fetch the first page.
+In the previous template string, the `cursor` is passed as variable to the query and used as `after` argument for the list field. The variable is not enforced though, because there is no exclamation mark next to it, so it can be `undefined`. This happens for the initial page requests for a paginated list where you only want to fetch the first page. Further, the argument `last` has been changed to `first` for the `issues` list field, because there won't be another page after you fetched the last item in the initial request. Thus, you have to start with the first items off the list to fetch more items until you reach the end of the list.
```

Comment:
"This happens for the initial page requests for a paginated list where you only want to fetch the first page." to "This happens for the initial page request for a paginated list where you only want to fetch the first page."

Reply:
"items off the list to fetch more items until" to "items of the list to fetch more items until"